WordPress/Classes/ViewRelated/Stats/JetpackSettingsViewController.m

```diff
@@ -95,6 +95,8 @@ - (BOOL)hidesBottomBarWhenPushed
 
 - (void)willAnimateRotationToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrientation duration:(NSTimeInterval)duration
 {
+    [super willAnimateRotationToInterfaceOrientation:toInterfaceOrientation duration:duration];
```

Comment:
It's my goal to replace this with a Storyboard or XIB since there are in fact three different UIs shoved into this view controller.